agent/header_times_streamer.go

```diff
@@ -73,18 +73,16 @@ func (h *HeaderTimesStreamer) Scan(line string) {
 	h.scanWaitGroup.Add(1)
 	defer h.scanWaitGroup.Done()
 
-	if h.lineIsHeader(line) {
```

Comment:
I'm probably missing something, but I think this should still be there? Otherwise each line is probably considered a `---header` right?